content/api/events.apib

```diff
@@ -0,0 +1,179 @@
+FORMAT: 1A
+title: Events
+description: Query detailed SparkPost event data for futher analysis and processing.
+
+# Group Events
+
+The Events API allows searching for recent events, and supports various types of result filtering. 
+Available events include message status – injection, delivery, bounce – as well as recipient engagement – clicks and opens. 
+You can fetch all event types or only specific ones, such as bounces, deliveries, or clicks.
+You can also filter by date range, campaign, or just about any other field.
+It provides access to similar event data that can also delivered via [Webhooks](/api/webhooks/).
+
+*Note:* The maximum URI length is 4096 characters, so please keep your queries below that size when adding filters to an API call.
+
+### Data Retention
+
+Data in the Events API is retained for 10 days. Aggregate reporting data is available through [metrics](/api/metrics/) or the app for up to 6 months.
+
+### Event Types
+
+You can query the following types of events from the Events API.
+
+<MessageEvents></MessageEvents>
+
+___
+
+__The SparkPost Events API uses MaxMind software [MaxMind License](https://www.maxmind.com/download/geoip/database/LICENSE.txt)__
+
+___
+
+### Search for Message Events [GET /v1/events/message{?recipients,from,to,timezone,cursor,per_page,events,subaccounts,campaign_ids,template_ids,message_ids,transmission_ids,from_addresses,bounce_classes,reason,subject,sending_domains,sending_ips,ip_pools,ab_test_ids,ab_test_version}]
+
+Returns a list of message events that matched the filtered search. The response is sorted by descending timestamp.
+
++ Parameters
+    + from (string, optional) - Datetime in format of YYYY-MM-DDTHH:MM.
+        + Default: `1 minute ago`
+    + to (string, optional) - Datetime in format of YYYY-MM-DDTHH:MM.
+        + Default: `24 hours ago`
+    + timezone (string, optional) - Standard timezone identification string.
+        + Default: `UTC`
+    + cursor (string, optional) - Results cursor for pagination. Used in conjunction with `per_page` parameter.
+        + Default: `initial`
+    + per_page (number, optional) - Maximum number of results to return per page. Must be between 1 and 10,000.
+        + Default: `1000`
+    + events (string, optional) - Comma delimited list of event types to search. Defaults to all event types.
+    + subaccounts (string, optional) - Comma delimited list of subaccount IDs to search.
+    + recipients (string, optional, `recipient@example.com`) ... Comma delimited list of recipients to search. Supports wildcarding in both the localpart and domain part of the email address. For example, `*@gmail.com` and `sparky@yahoo.*` are both valid values.
```

Comment:
new query parameter: recipient_domain, supports `yahoo` and `yahoo.jp`. `yahoo` will return ALL yahoo domains, `yahoo.jp` will return just `yahoo.jp` events

Reply:
NOT comma delimited list, can only support one field